src/Streamly/Internal/FS/Event/Linux.hs

```diff
@@ -759,9 +730,9 @@ removeFromWatch (Watch handle wdMap) path = do
     where
 
     step fd newMap (wd, v) = do
-        if fst v == path
+        if Path.toChunk (getRepRoot v) == Path.toChunk root
```

Comment:
Is `Path` gauaranteed to be normalized? What if one path is `a/b` and the other is `a//b`? Should we handle that? If we should then at least we can add a TODO comment here.

Reply:
See: https://github.com/composewell/streamly/issues/2888